### ISSUE TITLE
No data found. Please verify arguments

### ISSUE THREAD

**Issue by sashasubbbb** [kohya-ss/sd-scripts#20]
Hi, i don't really get why i'm getting the message

> No data found. Please verify arguments / 画像がありません。引数指定を確認してください

Did the 

> .\venv\Scripts\activate

Inside sd-scripts folder

Then 

> accelerate launch --num_cpu_threads_per_process 8 train_network.py --pretrained_model_name_or_path=B:\AIimages\stable-diffusion-webui\models\Stable-diffusion\model.ckpt --train_data_dir=B:\AIimages\training\images\input\ --output_dir=B:\AIimages\sd-scripts\output\ --prior_loss_weight=1.0 --resolution=512,512 --train_batch_size=4 --learning_rate=1e-4 --max_train_steps=200 --use_8bit_adam --xformers --mixed_precision=fp16 --save_every_n_epochs=1 --save_model_as=safetensors --clip_skip=1 --seed=42 --color_aug --network_module=networks.lora --unet_lr=5e-4 --text_encoder_lr=5e-5

There are 61 .pngs with .caption files inside input folder
